fedora/fas.py

```diff
@@ -217,3 +217,4 @@ async def aliases(self, evt: MessageEvent, match) -> None:
             await self._user_info(evt, arguments)
         elif cmd in ["localtime"]:
             await self._user_localtime(evt, arguments)
+        pass
```

Comment:
So, coverage complains here:

`line 218 didn't return from function 'aliases', because the condition on line 218 was never false` 

if i add the pass, it goes away. Is there a better way here?